src/components/containers/notification/Notification.js

```diff
@@ -146,6 +146,13 @@ function renderCallsToAction(callsToAction) {
       {callsToAction.map((callToAction, index) => {
         const buttonStyleType = index === 0 ? 'primary' : 'default';
 
+        if (React.isValidElement(callToAction)) {
```

Comment:
Started #73 for this discussion. I fully agree that the CTAs should be ripped out, and I do not want to introduce those changes here since it would necessitate a large swathe of changes downstream.